wherehows-backend/app/actors/ConfigUtil.java

```diff
@@ -63,7 +64,7 @@ static ProcessBuilder buildProcess(String etlJobName, long whEtlExecId, String c
       Properties etlJobProperties) {
     String classPath = System.getProperty("java.class.path");
     String outDir = etlJobProperties.getProperty(Constant.WH_APP_FOLDER_KEY, WH_APPLICATION_DEFAULT_DIRECTORY);
-    String configFile = outDir + "/exec/" + whEtlExecId + ".properties";
+    String configFile =  outDir + "/" + whEtlExecId + ".properties";
```

Comment:
Drop the extra space?

Reply:
Reformatted the code now which removed extra space.